client/state/reader/tags/images/test/actions.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * External dependencies
+ */
+import sinon from 'sinon';
+import { assert, expect } from 'chai';
+import deepFreeze from 'deep-freeze';
+
+/**
+ * Internal dependencies
+ */
+import useNock from 'test/helpers/use-nock';
+import {
+	READER_TAG_IMAGES_REQUEST,
+	READER_TAG_IMAGES_REQUEST_SUCCESS,
+	READER_TAG_IMAGES_RECEIVE,
+} from 'state/action-types';
+
+import { sampleSuccessResponse } from '../sample_responses';
+
+describe( 'actions', () => {
+	let receiveTagImages, requestTagImages;
+	const spy = sinon.spy();
+
+	beforeEach( () => {
+		spy.reset();
+		const actions = require( '../actions' );
```

Comment:
usuallly that's because the thing being require'd is using things being mocked out with mockery and we have to set up the mocks before loading the module. In this case though, I don't think there's a need.